index.html

```diff
@@ -0,0 +1,124 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>판다마켓</title>
+
+  <link rel="stylesheet" href="index.css"/>
+  <link rel="stylesheet" as="style" crossorigin href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.8/dist/web/static/pretendard.css" />
+
+</head>
+<body>
+  <header>
+    <a class="logo" href="/">
+      <img class="logo-img" src="img/logo.svg">
+      <img class="logo-title" src="img/logo-title.svg">
+    </a>
+    
+    <div class="login">
+      <a class="login-btn" href="/login.html">로그인</a>
+    </div>
+  </header>
+
+  <section class="outside top">
+    <div class="top-contents">
+      <div class="top-wording">
+        <h2 class="top-wording-title">
+          일상의 모든 물건을<br>
+          거래해 보세요
+        </h2>
+        <a class="top-wording-btn" href="/items.html">구경하러 가기</a>
+      </div>
+  
+      <img class="top-img" src="img/outside-top.svg">
+    </div>
+  </section>
+
```

Comment:
```suggestion
<main>  
  <section class="outside top">
    <div class="top-contents">
      <div class="top-wording">
        <h2 class="top-wording-title">
          일상의 모든 물건을<br>
          거래해 보세요
        </h2>
        <a class="top-wording-btn" href="/items.html">구경하러 가기</a>
      </div>
  
      <img class="top-img" src="img/outside-top.svg">
    </div>
  </section>
</main>

```

위 처럼 `main` 태그를 사용해서 section들을 전부 감싸는 시멘틱 태그도 사용해보면 좋겠어요.